Source/Extensions/Blazorise.RichTextEdit/JSRichTextEditModule.cs

```diff
@@ -34,25 +29,32 @@ public JSRichTextEditModule( IJSRuntime jsRuntime, IVersionProvider versionProvi
         : base( jsRuntime, versionProvider )
     {
         this.options = options;
+        ModuleLoaded = OnModuleLoaded;
     }
 
     #endregion
 
     #region Methods
+    private async ValueTask OnModuleLoaded( IJSObjectReference @ref )
+    {
+        List<string> styles = new();
+        if ( options.UseBubbleTheme )
+            styles.Add( "bubble" );
+        if ( options.UseBubbleTheme )
+            styles.Add( "snow" );
+
+        if ( styles.Count > 0 )
+        {
+            await @ref.InvokeVoidAsync( "loadStylesheets", string.Join( ',', styles ), VersionProvider.Version );
```

Comment:
Instead of csv we can have an array of strings.

Reply:
ok I will change that